acra-censor/handlers/blacklist_handler.go

```diff
@@ -105,8 +103,9 @@ func (handler *BlacklistHandler) CheckQuery(query string) (bool, error) {
 func (handler *BlacklistHandler) handleAliasedTables(statement *sqlparser.AliasedTableExpr) error {
 	if handler.tables[sqlparser.String(statement.Expr)] {
 		return ErrAccessToForbiddenTableBlacklist
+	} else {
```

Comment:
golint has a rule: if method ends on 

```
if 
   return
else 
   return
```

it should be changed to (remove else)

```
if 
  return

return
```